crypto/cmp/cmp_ctx.c

```diff
@@ -590,7 +590,7 @@ int OSSL_CMP_CTX_set1_##FIELD(OSSL_CMP_CTX *ctx, const TYPE *val) \
     \
     if (val != NULL && (val_dup = TYPE##_dup(val)) == NULL) \
         return 0; \
-    TYPE##_free(ctx->FIELD); \
+    TYPE##_free((TYPE *)ctx->FIELD);                  \
```

Comment:
This is really suspicious. The FIELD should not be const then.

Reply:
See above